app/msg/[username]/page.js

```diff
@@ -31,7 +44,10 @@ function SendMessage(id) {
           rows={6}
         ></textarea>
       </div>
-      <button className=" font-bold px-4 py-2 rounded-lg bg-[wheat]">
+      <button
+        onClick={() => fetchMessage()}
```

Comment:
[optional] if no argument/parameter is to be passed in an onClick handler function, much cleaner way to write would be :
```
<button onClick = {handlerFunctionName} ... />
```

Reply:
Great suggestion as passing the arrow function incline creates function on each render , will do the changes ,
thanks